x/emissions/types/errors.go

```diff
@@ -1,14 +1,13 @@
 package types
 
-// DONTCOVER
+import errorsmod "cosmossdk.io/errors"
 
-import (
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-)
+// DONTCOVER
 
-// x/emissions module sentinel errors
 var (
-	ErrEmissionTrackerNotFound = sdkerrors.Register(ModuleName, 1100, "Emission Tracker Not found")
-	ErrParsingSenderAddress    = sdkerrors.Register(ModuleName, 1101, "Unable to parse address of sender")
-	ErrAddingCoinstoTracker    = sdkerrors.Register(ModuleName, 1102, "Unable to add coins to emissionTracker ")
+	ErrEmissionsNotFound                   = errorsmod.Register(ModuleName, 1000, "Emissions not found")
+	ErrUnableToWithdrawEmissions           = errorsmod.Register(ModuleName, 1002, "Unable to withdraw emissions")
+	ErrInvalidAddress                      = errorsmod.Register(ModuleName, 1003, "Invalid address")
+	ErrRewardsPoolDoesNotHaveEnoughBalance = errorsmod.Register(ModuleName, 1004, "Rewards pool does not have enough balance")
+	ErrInvalidAmount                       = errorsmod.Register(ModuleName, 1005, "Invalid amount")
```

Comment:
We should use full lower case for errors

Reply:
f2d4d84768a74f837eece318f4b2c9fa557c3f81